css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,31 @@
+* {
+  font-family: "DM Sans", sans-serif;
+  font-weight: 400;
+}
+
+.col-full {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  margin: 0 5%;
+}
+
+.col-fifth {
+  flex-basis: calc(100% / 5);
+  margin: 0 2%;
+}
+
+img {
+  width: 250px;
+  height: 450px;
+}
```

Comment:
You don't want to set a fixed width AND height, because it screws up the aspect ratio of the images. If you're going to use a set px value, you should only set one. Something like
```
img {
  width: 100%;
  height: 400px;
}